modules/flowable-bpmn-model/src/main/java/org/flowable/bpmn/model/AbstractFlowableHttpHandler.java

```diff
@@ -14,17 +14,20 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
```

Comment:
I don't see `Objects` used in this class.

Reply:
fixed